storage/blockchain/src/ops/mod.rs

```diff
@@ -55,25 +55,28 @@
 //! use hex_literal::hex;
 //!
 //! use cuprate_test_utils::data::block_v16_tx0;
-//!
 //! use cuprate_blockchain::{
-//!     ConcreteEnv,
+//!     cuprate_database::{
+//!         ConcreteEnv,
+//!         Env, EnvInner,
+//!         DatabaseRo, DatabaseRw, TxRo, TxRw,
+//!     },
+//!     OpenTables,
 //!     config::ConfigBuilder,
-//!     Env, EnvInner,
 //!     tables::{Tables, TablesMut},
-//!     DatabaseRo, DatabaseRw, TxRo, TxRw,
 //!     ops::block::{add_block, pop_block},
 //! };
```

Comment:
@Boog900 Lots of the `cuprate_database` traits are needed for `cuprate_blockchain::ops`.

If we scoped the required imports, it looks like:
```rust
pub use cuprate_database::{
    ConcreteEnv, Env, EnvInner,
    InitError, RuntimeError,
    DatabaseRo, DatabaseRw, DatabaseIter,
    TxRo, TxRw
};
```
...that's pretty much everything :D